### PR TITLE
RCBC-297 - Named Scope and Collections and 3.2/7.0 updates

### PR DESCRIPTION
**Pull request by maria-robobug**

### CODE REVIEW

File: modules/concept-docs/examples/collections.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```
Comment (osfameron) [inline modules/concept-docs/examples/collections.rb:1]: ☃️

File: modules/concept-docs/pages/collections.adoc [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+And for a non-default scope:
+
+[source,ruby]
+----
+include::example$collections.rb[tag=collections_2,indent=0]
+----
+
+== Further Reading
+
+To see Collections in action, take a look at our xref:howtos:working-with-collections.adoc[Collections-enabled Travel Sample page].
```
Comment (osfameron) [inline modules/concept-docs/pages/collections.adoc:33]: Hmm, this isn't what we're doing anymore is it? We should point this to sample-application.adoc instead.

(Eeek, I've written a similar line before... it's maybe ok-for-now, as there'll be just one layer of indirection as the working-with-collections page will then point to travel-sample, but we should fix, today if poss, or this week)

File: modules/concept-docs/pages/encryption.adoc [hunk 3]
```diff
@@ -11,4 +11,3 @@ include::partial$attributes.adoc[]
 
 Client-side implementation of Field Level Encryption is available in previous versions of the C, .NET, Go, Java, Node.js, PHP, and Python SDK.
 It will be enabled in a future release of the third generation SDK, including the Ruby SDK.
```
Comment (osfameron) [inline modules/concept-docs/pages/encryption.adoc:13]: might be worth commenting https://issues.couchbase.com/browse/RCBC-284

File: modules/hello-world/pages/sample-application.adoc [hunk 4]
```diff
@@ -99,7 +99,7 @@ The https://github.com/couchbaselabs/try-cb-nodejs/blob/6.5/index.js[index.js] f
 ////
 
 
-NOTE: Travel Sample app is not currently available for the recently-released 3.0 Couchbase Ruby SDK.
+NOTE: Travel Sample app is not currently available for the recently-released 3.2 Couchbase Ruby SDK.
```
Comment (osfameron) [inline modules/hello-world/pages/sample-application.adoc:102]: OK, yes, that was easier than I was expecting :-)

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 5]
```diff
@@ -65,23 +66,24 @@ To access the KV (Key/Value) API or to query views, you need to open a `Bucket`:
 include::example$start_using.rb[tag=bucket,indent=0]
 ----
 
-// If you installed the travel sample data bucket, substitute _travel-sample_ for _bucket-name_.
+If you installed the travel sample data bucket, substitute _travel-sample_ for _bucket_name_.
```
Comment (osfameron) [inline modules/hello-world/pages/start-using-sdk.adoc:69]: did you verify that the markup (e.g. italics vs underscore) work correctly here, I can't remember how antora handles this, but has bitten me with other systems in the past...

**Comment by osfameron** [comment]
LGTM! 👍

File: modules/concept-docs/pages/collections.adoc (hunk 2 above)
Comment (maria-robobug) [reply modules/concept-docs/pages/collections.adoc:33]: Think I will comment this out, we don't even have a travel-sample Ruby app yet >_<

File: modules/hello-world/pages/start-using-sdk.adoc (hunk 5 above)
Comment (maria-robobug) [reply modules/hello-world/pages/start-using-sdk.adoc:69]: Yep works fine 👍

**Pull request merged**